EnglishFilter/sections/specific.txt

```diff
@@ -263,6 +263,8 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+||kittyglitter-*/brand/specbonus-image/bet365/*.png
```

Comment:
please gise url of the page. Cannot find where these rules can be applied https://github.com/AdguardTeam/AdguardFilters/pull/78967/files#diff-8e737f6429801c4aac12f0a8ddb7b4ab3173f39642bfe56004e22dede37edd6dR266-R267

scriptlet is ok

Reply:
`@stanislav-atr` added it. I have two more rules which he has missed.

Reply:
@Alex-302 most of changes was merged from https://github.com/AdguardTeam/AdguardFilters/pull/78509.

Reply:
<details> 

![image](https://user-images.githubusercontent.com/76908410/113133898-c67d3500-9252-11eb-9439-43e9b0e476cc.png)
</details>

Reply:
@zloyden @Alex-302 He missed two things. I have told him one rule at his PR, and yet he ignored/denied it.

Reply:
Not reproduced on my end, but I'd add `||amazonaws.com/brand/$domain=hltv.org`.

Reply:
Yeah, I agree.